dockerfiles/Dockerfile.jetson

```diff
@@ -0,0 +1,36 @@
+# syntax=docker/dockerfile:experimental
+#
+# This Dockerfile just installs pre-built ONNXruntime wheel inside the image.
+# Please make sure you have nvidia-runtime enabled in docker config and then build like:
+#
+# sudo -H DOCKER_BUILDKIT=1 nvidia-docker build --build-arg WHEEL_FILE=<path> -f Dockerfile.jetson
+#
+
+ARG BASE_IMAGE=nvcr.io/nvidia/l4t-base:r32.4.3
+FROM ${BASE_IMAGE} as onnxruntime
+
+ARG WHEEL_FILE
+RUN echo "Building ONNXruntime Docker image with ${WHEEL_FILE}..."
+
+# Ensure apt-get won't prompt for selecting options
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt update && \
+    apt install -y --no-install-recommends \
+    	build-essential \
+	software-properties-common \
+	libopenblas-dev \
+        libpython3.6-dev \
+        python3-pip \
+        python3-dev \
+	cmake
+
+RUN pip3 install --upgrade pip
+RUN pip3 install setuptools wheel pybind11 pytest
```

Comment:
When executing this command on my Jetson Xavier with Jetpack 4.2 and docker version 18.09.7. 
I got following error:

ERROR [5/7] RUN pip3 install setuptools wheel pybind11 pytest                                                                                                                                                                                                                   73.5s
------
 > [5/7] RUN pip3 install setuptools wheel pybind11 pytest:
#13 17.24 Collecting setuptools
#13 23.99   Downloading setuptools-49.2.0-py3-none-any.whl (789 kB)
#13 43.04 Collecting wheel
#13 44.73   Downloading wheel-0.34.2-py2.py3-none-any.whl (26 kB)
#13 46.44 Collecting pybind11
#13 47.62   Downloading pybind11-2.5.0-py2.py3-none-any.whl (296 kB)
#13 53.03 Collecting pytest
#13 54.42   Downloading pytest-6.0.0-py3-none-any.whl (270 kB)
#13 56.39 Collecting pluggy<1.0,>=0.12
#13 57.59   Downloading pluggy-0.13.1-py2.py3-none-any.whl (18 kB)
#13 59.25 Collecting py>=1.8.2
#13 60.47   Downloading py-1.9.0-py2.py3-none-any.whl (99 kB)
#13 62.17 Collecting more-itertools>=4.0.0
#13 63.10   Downloading more_itertools-8.4.0-py3-none-any.whl (43 kB)
#13 64.33 Collecting importlib-metadata>=0.12; python_version < "3.8"
#13 65.53   Downloading importlib_metadata-1.7.0-py2.py3-none-any.whl (31 kB)
#13 66.81 Collecting iniconfig
#13 67.91   Downloading iniconfig-1.0.0.tar.gz (7.8 kB)
#13 68.04     ERROR: Command errored out with exit status 1:
#13 68.04      command: /usr/bin/python3 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-e6451fdx/iniconfig/setup.py'"'"'; __file__='"'"'/tmp/pip-install-e6451fdx/iniconfig/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-pip-egg-info-yfaxxd5w
#13 68.04          cwd: /tmp/pip-install-e6451fdx/iniconfig/
#13 68.04     Complete output (3 lines):
#13 68.04     Traceback (most recent call last):
#13 68.04       File "<string>", line 1, in <module>
#13 68.04     ModuleNotFoundError: No module named 'setuptools'
#13 68.04     ----------------------------------------
#13 68.04 ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
------